packages/ocamline/ocamline.1.1/opam

```diff
@@ -0,0 +1,22 @@
+opam-version: "2.0"
+synopsis: "Command line interface for user input"
+maintainer: "Chris Nevers <christophernevers96@gmail.com>"
+authors: "Chris Nevers <christophernevers96@gmail.com>"
+homepage: "https://github.com/chrisnevers/ocamline"
+bug-reports: "https://github.com/chrisnevers/ocamline/issues"
+depends: [
+  "dune" {build}
```

Comment:
```suggestion
  "dune" {>= "1.9"}
```
Dune should not be tagged as a build dependency (see #12466). Moreover, according to your `dune-project` file, this package requires at least dune 1.9